packages/gatsby-plugin-netlify/README.md

```diff
@@ -34,6 +34,7 @@ plugins: [
       mergeCachingHeaders: true, // boolean to turn off the default caching headers
       transformHeaders: (headers, path) => headers, // optional transform for manipulating headers under each path (e.g.sorting), etc.
       generateMatchPathRewrites: true, // boolean to turn off automatic creation of redirect rules for client only paths
+      contentHashes: false, // boolean to remove content hashes for assets generated
```

Comment:
- should not be in this first example the default here: `true`, 
(because later it is explained when to change it to `false`) 
- and also add an `optional`-notation to it?

Reply:
Have changed these to your recommendations. Wasn't aware that was the convention.

Reply:
See new comment